### DISCUSSION TITLE
My HA dashbaord

### DISCUSSION THREAD

**Discussion by pdhruv93** [shannonhochkins/ha-component-kit#41] [Show and tell]
It was quite easy to use the components(new UI cards coming) and deploy the final build to HA. Not to forget the developer who is super-active in maintaining the libraries, any feature request is instantly acknowledged.
**This is not a sponsored post and I would like to share the dashboard that I created myself. I have also shared it to the Facebook group for Home Assistant and hopefully more people notice this.
![image](https://github.com/shannonhochkins/ha-component-kit/assets/38987871/69325bcf-52c2-471c-8ce0-76e87f4e33a9)

**Comment by shannonhochkins** [comment]
You're a legend - thanks buddy! Looks great :) I've got some cool plans for this project to make the dashboards look & feel nicer, and really appreciate the compliment :)

Would love a link to the thread on FB if you have it so i can have a read!

**Reply by pdhruv93** [reply]
Here is the [link](https://www.facebook.com/groups/HomeAssistant/permalink/3578453889092622/)
Please let me know if you are part of this HA group, it's very good. If not you can send me your FB username, I will invite you there.

**Reply by shannonhochkins** [reply]
Thanks legend, i'm actually not part of that group - https://www.facebook.com/therealshannonnoll
